### ISSUE TITLE
[Item] Can't equip belt anymore after restart.

### ISSUE THREAD

**Issue by Sugoiidesune** [dalaranwow/dalaran-wow#5823]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Belt item [Belt of the Betrayed] glitched when I logged in after server restart.
It unequipped itself, arrived in a 'error equipping this item' mail, and now can't equip by any means.
**Expected behaviour**: Tell us what should happen instead
Belt should be equipable
**Steps to reproduce the problem**:
unknown
1. 
2. 
3. 

**Include proofs for this behaviour**
Item id: 45467

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
![WoWScrnShot_101219_213547](https://user-images.githubusercontent.com/56493737/66709750-98dbbd80-ed38-11e9-986e-9b0b3d07bff9.jpg)

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Chenocide** [comment]
Do you have a Nightmare Tear inserted in another piece of equipment?

**Comment by Golganeth** [comment]
Nightmare Tear is Unique-Equipped, which means you can put Nightmare Tears in as many gear parts as you want, but you will only be able to equip one of those at once. 

You are probably trying to wear that belt while already having a Nightmare Tear active in your equipment. If that's the case, it's totally intended. If not, it could be an isolated issue so please contact us through the in-game ticket and we will check it out.
